src/App.js

```diff
@@ -1,10 +1,25 @@
 import './App.css';
 import React from 'react';
+import { Route, Routes } from 'react-router-dom';
 import Calculator from './components/Calculator';
-
-class App extends React.PureComponent {
+import Home from './components/Home';
+// eslint-disable-next-line import/extensions
+import NavigationBar from './components/NavigationBar';
+import Quote from './components/Quote';
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
### OPTIONAL


- [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks 👍🏿

Reply:
@prolajumokeoni, thanks for the Code Review and the advice on Linters.